### PR TITLE
Fixed the documentation of the quit menu items

### PR DESCRIPTION
**Pull request by omichel**
Dual PR with https://github.com/omichel/webots/pull/7052

### CODE REVIEW

**Comment by omichel** [comment]
👍

File: guide/the-user-interface.md [hunk 1]
```diff
@@ -89,9 +89,13 @@ You can get more information about this topic in [this section](web-scene.md).
 Once the animation recording is started, this item is changed to `Stop HTML5 Animation...` and can be used to stop the animation recording.
 You can get more information about this topic in [this section](web-animation.md).
 
-- **Logout and quit Webots** terminates the current simulation, closes Webots and resets the user credentials so that at the next Webots startup the login dialog pops up requesting the user's e-mail and password.
+- **Logout and Quit** terminates the current simulation, closes Webots and resets the user credentials so that at the next Webots startup the login dialog pops up requesting the user's e-mail and password. This menu item is available only when being logged-in with a user account.
```
Comment (stefaniapedrazzi) [inline guide/the-user-interface.md]: I'm not sure about: "when being logged-in".
To me it sounds better: "when logged-in".

Comment (omichel) [reply guide/the-user-interface.md]: Yes, that sounds better to me as well. I fixed it. Can you re-approve so that I can merge it?

**Comment by omichel** [comment]
👍

**Pull request merged**